variables/_svgs.pcss

```diff
@@ -120,6 +131,24 @@
 	}
 }
 
+/* -----------------------------------------------------------------------------
+ * SVG - Close
+ * ----------------------------------------------------------------------------- */
+
+@svg-load close url('../icons/close.svg') {
+
+	path {
+		stroke: var(--color-icon-secondary);
```

Comment:
`close` should use primary color and `close-secondary` should use secondary color, keeping naming consistent with color we use.